src/components/player-controls.ts

```diff
@@ -45,6 +55,13 @@ class PlayerControls extends LitElement {
             ></mxmp-volume>
           `,
         )}
+        ${when(
+          ['off'].includes(this.activePlayer.state),
+          () => html`
+            <mxmp-ha-player hide=${showPowerButton} .store=${this.store} .features=${[TURN_ON, TURN_OFF]} ></mxmp-ha-player>
```

Comment:
Power button is already in the volume.ts, I don't think it is needed here as well.

Reply:
The volume controls are not shown at that point (at least in my configuration), so no power button is shown without this code.  All I see is a big musical note and a `No Media Selected` title.

Reply:
Ah, makes sense.
Could you maybe incorporate `['off'].includes(this.activePlayer.state),` into `showPowerButton`?

Reply:
> Could you maybe incorporate `['off'].includes(this.activePlayer.state),` into `showPowerButton`?

That cannot be done easily, as the `hide={$xxxx}` attribute value is not a True or False value.  The `hide` attribute is removed from the parent element if the hide attribute value is `undefined`.  I spent almost 10 hours looking into that, and finally realized how that worked.  It's very misleading, in that I thought the attribute value was generated as True (meaning hide the element) or False (show the element).  The hide attribute value can be ANY non-null value, and it will hide the element.